File: _toolkit/-JS-My-Functions/shuffle.js
//shuffle an array
function shuffle(o){
    for(var j,x,i = o.length; i; j=Math.floor(Math.random()*i), x=o[--i], o[i] = o[j], o[j] = x);
    return o;
}
y = shuffle([2,3,4,5,6,7,8,9,"j","q","k","a"]);

File: python/21/toolkit/shuffle.py
from random import randint as rand

def shuffles(times, deck):
    shuffled = deck
    for i in range(times):
        shuffled = shuffle(shuffled)
    return shuffled

def shuffle(deck):
    shuffled = []
    while len(deck) > 0:
        drawIndex = rand(0,len(deck)-1)
        shuffled.append(deck[drawIndex])
        del deck[drawIndex]
    return shuffled
